src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -148,7 +148,8 @@ public string GetActiveBufferName()
 
         public void WaitForActiveView(string expectedView)
         {
-            Retry(GetActiveBufferName, (actual) => actual == expectedView, TimeSpan.FromMilliseconds(100));
+            using var cts = new CancellationTokenSource(Helper.HangMitigatingTimeout);
+            Retry(_ => GetActiveBufferName(), (actual, _) => actual == expectedView, TimeSpan.FromMilliseconds(100), cts.Token);
```

Comment:
What is the benefit of changing `GetActiveBufferName` to `_ => GetActiveBufferName()`?

Reply:
The signature no longer matches because `GetActiveBufferName` doesn't take the `CancellationToken`.